test/testlib/nuodb_database_utilities.go

```diff
@@ -383,7 +383,9 @@ func GetNuoDBVersion(t *testing.T, options *helm.Options) string {
 }
 
 func SkipTestOnNuoDBVersion(t *testing.T, versionCheckFunc func(*semver.Version) bool) {
-	versionString := GetNuoDBVersion(t, &helm.Options{})
+	randomSuffix := strings.ToLower(random.UniqueId())
+	namespaceName := fmt.Sprintf("nuodbversioncheck-%s", randomSuffix)
+	versionString := GetNuoDBVersion(t, namespaceName, &helm.Options{})
```

Comment:
do you have to delete the namespace after?

Reply:
This should be done by teardown placed in `CreateNamespace` function.